assets/css/membership.css

```diff
@@ -46,22 +46,6 @@
     font-size: 23px;
 }
 
-.membership-page .plan-price .mb-0 b{
-    font-size: 32px;
-    font-weight: 700;
-    line-height: 1.2;
-    letter-spacing: 0.03rem;
-    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
-}
-
-.membership-page .plan-price .mb-0 b{
-    font-size: 32px;
-    font-weight: 700;
-    line-height: 1.2;
-    letter-spacing: 0.03rem;
-    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
-}
-
```

Comment:
If you remove both CSS classes you will lose style for prices on membership page:

With style:
![Screenshot 2021-09-29 at 20 23 42](https://user-images.githubusercontent.com/12160621/135318445-56881dc5-b7fe-4c7f-b5f7-dbb41693367f.png)

Without style:
![Screenshot 2021-09-29 at 20 23 52](https://user-images.githubusercontent.com/12160621/135318479-5a5e25b6-34a7-400f-804e-9434deb9b7d4.png)